providers/hns/hns_roce_u_hw_v2.h

```diff
@@ -40,9 +40,10 @@
 
 #define HNS_ROCE_CMDSN_MASK			0x3
 
-enum {
-	HNS_ROCE_SUPPORT_RQ_RECORD_DB = 1 << 0,
-	HNS_ROCE_SUPPORT_SQ_RECORD_DB = 1 << 1,
+enum hns_roce_qp_flags {
```

Comment:
Why aren't those flags in hns-abi.h? They exist in the kernel and taken from there.

Reply:
Thanks for your reminder. These flag should be in hns-abi.h, I will modify the code.